### DISCUSSION TITLE
Upgrade to Mermaid 10.x

### DISCUSSION THREAD

**Discussion by senpro-ingwersenk** [outline/outline#7228] [Ideas]
Hello!

Really simple, really short. Aside from CJS removal in [Mermaid 10.0.0](https://github.com/mermaid-js/mermaid/releases/tag/v10.0.0), what else would need to be done to migrate to it? We would like to use some of the newer features introduced there in our internal documentation.

I had considered just doing the PR for this myself, but before I sink some time into building and testing, I wanted to ask back beforehand in case a plan for this was already in place.

Thank you and kind regards!

**Comment by tommoor** [comment]
As long as it:

- works correctly
- doesn't increase the _initial_ bundle size
- doesn't depend on loading from a third party CDN
 
…that's all that's required, unfortunately no PR so far has managed to achieve that partly because of the CJS removal
